runtime/oti/jvminit.h

```diff
@@ -418,6 +418,9 @@ enum INIT_STAGE {
 #define VMOPT_XXENABLEHCR "-XX:+EnableHCR"
 #define VMOPT_XXNOENABLEHCR "-XX:-EnableHCR"
 
+#define VMOPT_XXUSECONTAINERSUPPORT "-XX:+UseContainerSupport"
+#define VMOPT_XXDONTUSECONTAINERSUPPORT "-XX:-UseContainerSupport"
```

Comment:
`VMOPT_XXDONTUSECONTAINERSUPPORT` -> `VMOPT_XXNOUSECONTAINERSUPPORT`

I realize the grammar of the naming convention is quiet poor, but at least it's consistent :)